src/app.ts

```diff
@@ -38,6 +39,7 @@ export function generateServerlessRouter(
     const serverUrl: string = fhirConfig.server.url;
     let hasCORSEnabled: boolean = false;
     const registry = new FHIRStructureDefinitionRegistry(compiledImplementationGuides);
+    const operationRegistry = initializeOperationRegistry(configHandler);
```

Comment:
This is probably too meta/another PR but could/should we use this for existing operations i.e. 'create', 'search', etc?